sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsAudience.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary> Cloud audiences available for TextAnalytics. </summary>
+    public readonly partial struct TextAnalyticsAudience : IEquatable<TextAnalyticsAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TextAnalyticsAudience"/> object.
```

Comment:
We're missing a `<param>` tag for `value`.    We should detail that the value for an audience should be a URL similar to  "https://cognitiveservices.azure.com"